portfolio/src/main/webapp/comments.js

```diff
@@ -32,7 +32,7 @@ function renderCommentsChart(){
     table.addColumn('number', 'Count');
     
     json.forEach((word) => {
-      table.addRow([word.key.name, word.propertyMap.count]);
+      table.addRow([word.key.name.substring(0, word.key.name.length - 5), word.propertyMap.count]);
```

Comment:
What's 5

Reply:
The length of the end substring l want to remove. The key name for "cat" would be cat_word. l just want "cat".

Reply:
name the constant?

Reply:
keySuffixLen = 5 is probably fine, though consider including comment "suffix is '_word'"